bin/tag-release-start.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+const commander = require("commander");
+const { extend } = require("lodash");
+const api = require("../src/index.js");
+const workflow = require("../src/workflows/default");
+
+commander
+	.option("--verbose", "Console additional information")
+	.option(
+		"--maxbuffer <n>",
+		"Overrides the max stdout buffer of the child process. Size is 1024 * <n>.",
+		parseInt
+	)
+	.option(
```

Comment:
I thought we were going to remove this option, right?

Reply:
We can remove it.